pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -4191,3 +4073,14 @@ func (alc *fakeAllocator) Allocate(_ *virtv1.VirtualMachineInstance) error {
 func (alc *fakeAllocator) Remove(key string) {
 	alc.calls = append(alc.calls, "Remove")
 }
+
+type stubNetStatusUpdater struct {
+	updatedStatus   bool
+	statusUpdateErr error
+}
+
+func (snsu *stubNetStatusUpdater) Update(_ *virtv1.VirtualMachineInstance, _ *k8sv1.Pod) error {
```

Comment:
nit: It is common to see one or two letters for receivers. I do not think there is much added value to reflect all capital letters here, an `s` or `n` should be enough.

I'm commenting on this because it pops up as an anomaly to my eyes, so it could be just me.

Reply:
The whole stub struct was removed.